mx.sulong/mx_sulong.py

```diff
@@ -1161,6 +1170,7 @@ def sulongBuild(args=None):
     'argon2' : runTestArgon2,
     'lifetime' : runLifetimeTestCases,
     'main-arg' : runMainArgTestCases,
+    'pipe' : runPipeTestCases,
```

Comment:
Maybe you can register this as a `mx su-tests` suite? Then we do not need an additional mx command for that.